components/Layout.js

```diff
@@ -32,6 +40,42 @@ export default function Layout({ children, meta }) {
           href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital,wght@0,400;0,700;1,400&family=Libre+Franklin:ital,wght@0,300;0,800;1,300;1,800&display=swap"
           rel="stylesheet"
         />
+        {/* Twitter Card data */}
+        <meta name="twitter:card" content="summary_large_image" />
+        <meta name="twitter:site" content="@publisher_handle" />
+        <meta name="twitter:title" content={metaValues.twitterTitle} />
+        <meta
+          name="twitter:description"
+          content={metaValues.twitterDescription}
+        />
+        <meta name="twitter:creator" content="@author_handle" />
+        <meta
+          name="twitter:image:src"
+          content="http://www.example.com/image.jpg"
+        />
+
+        {/* Facebook data */}
+        <meta property="og:title" content={metaValues.facebookTitle} />
+        <meta property="og:type" content="article" />
+        <meta property="og:url" content="http://www.example.com/" />
+        <meta property="og:image" content="http://example.com/image.jpg" />
+        <meta
+          property="og:description"
+          content={metaValues.facebookDescription}
+        />
+        <meta property="og:site_name" content="Site Name, i.e. Moz" />
+        <meta
+          property="article:published_time"
+          content="2013-09-17T05:59:00+01:00"
```

Comment:
Send the `firstPublishedOn` for this. You can use the straight ISO time, no need to parse it.